.gitignore

```diff
@@ -11,3 +11,6 @@ coverage.txt
 *.pprof
 /.bin
 /.cache
+/bin
```

Comment:
Is there a reason we've got a `/bin` and a `/.bin`? Might make sense to combine in a later clean up

Reply:
/bin because we use it in the Makefile now. `/.bin` probably a leftover from old copy-pasta. Will make an attempt to clean up gitignores in a future change.